examples/bank/bank.go

```diff
@@ -117,43 +122,36 @@ func (bank *Bank) continuousMoneyTransfer(cash int64) {
 			if err := runner.Run(batchRead); err != nil {
 				return err
 			}
-			if batchRead.Results[0].Err != nil {
-				return batchRead.Results[0].Err
-			}
+			var err error
 			// Read from value.
 			fromAccount := &Account{}
-			err := fromAccount.decode(batchRead.Results[0].Rows[0].ValueBytes())
-			if err != nil {
-				return err
-			}
-			// Ensure there is enough cash.
-			if fromAccount.Balance < exchangeAmount {
-				return nil
-			}
+			fromAccount.decode(batchRead.Results[0].Rows[0].ValueBytes(), &err)
 			// Read to value.
 			toAccount := &Account{}
-			errRead := toAccount.decode(batchRead.Results[0].Rows[1].ValueBytes())
-			if errRead != nil {
-				return errRead
-			}
+			toAccount.decode(batchRead.Results[0].Rows[1].ValueBytes(), &err)
 			// Update both accounts.
-			batchWrite := &client.Batch{}
 			fromAccount.Balance -= exchangeAmount
 			toAccount.Balance += exchangeAmount
-			if fromValue, err := fromAccount.encode(); err != nil {
-				return err
-			} else if toValue, err := toAccount.encode(); err != nil {
+			fromValue := fromAccount.encode(&err)
```

Comment:
eg: fromAccount may be the result of a failed "decode", but we're still going anyway.